### ISSUE TITLE
The "regular" form item is not working as expected

### ISSUE THREAD

**Issue by LeoAref** [GeekyAnts/NativeBase#582]
## react-native, react and native-base version
- react-native: `0.41.2`
- native-base": `^2.0.11`

## Expected behaviour
When using *regular* form *item* >> Have an *input* field with 4 borders

## Actual behaviour
Getting the normal input with just border-bottom

## Steps to reproduce (code snippet or screenshot)
```js
<Text style={this.props.style.label}>English:</Text>
<Item regular>
  <Input style={this.props.style.input} placeholder='Regular Textbox'/>
</Item>
```
<img width="307" alt="screen shot 2017-03-01 at 09 54 12" src="https://cloud.githubusercontent.com/assets/6577222/23452613/3af6fee2-fe65-11e6-86ab-2cb03b2efbe0.png">

## Screenshot of emulator/device
Nexus 6 

## Is the bug present in both ios and android or in any one of them?
Tried only on Android

## Any other additional info which would help us debug the issue quicker.
No

**Comment by andidev** [comment]
The docsalso mentions the bordered attribute but that does not seem to work as well.

**Comment by sellmeadog** [comment]
I'm experiencing the same issue in the iOS simulator.  The "rounded" border options also renders with just a bottom border.

**Comment by sankhadeeproy007** [comment]
The rounded issue is a bug right now and we'll fix it in the future releases. There is however no ETA for it. If anybody would like to fix it with a PR, it'd be great. I can always walk you through the code in case anybody is interested.

**Comment by LeoAref** [comment]
@sankhadeeproy007 I'd like to submit PR for it, I'll work on it in the next few days

**Comment by michaelimstepf** [comment]
I'm having the same issues. Is this related to #641 and PR #691 ?

**Comment by shivrajkumar** [comment]
Fixed in v2.1.3
